lib/beaker/host.rb

```diff
@@ -499,6 +501,7 @@ def do_rsync_to from_path, to_path, opts = {}
     'freebsd',
     'windows',
     'pswindows',
+    'eos',
```

Comment:
Oh yeah, another less-than-ideal way: not declaring the base class and just loading the subclasses at the top of the file, and just ensuring the declaration (all nested modules, etc.) is done exactly the same way everywhere.  If you get it wrong, you'll get an error, so it's not really risky.